modules/install-sno-requirements-for-installing-on-a-single-node.adoc

```diff
@@ -8,7 +8,11 @@
 
 Installing {product-title} on a single node alleviates some of the requirements for high availability and large scale clusters. However, you must address the following requirements:
 
-* *Administration host:* You must have a computer to prepare the ISO, to create the USB boot drive, and to monitor the installation.
+* *Administration host:* You must have a computer to prepare the ISO, to create the USB boot drive, and to monitor the installation. 
+[NOTE]
+====
+For the `ppc64le` platform, the host should create the ISO, but need not do the USB boot drive. The ISO can be mounted to PowerVM directly.
```

Comment:
```suggestion
For the `ppc64le` platform, the host should prepare the ISO, but does not need to create the USB boot drive. The ISO can be mounted to PowerVM directly.
```

Reply:
> The ISO can be mounted to PowerVM directly.
Does the `ppc641e` platform do this step, or does the user do this step?

Reply:
Users have to do it.